src/components/BookList.js

```diff
@@ -1,21 +1,101 @@
 import Book from './Book'
+import styled from 'styled-components/macro'
+import { useState } from 'react'
 
 function BookList({ books }) {
+  const [filteredBooks, setFilteredBooks] = useState(books)
+  const [readingStatus, setReadingStatus] = useState('')
+
+  // for filtering and displaying book data (currently read & finished)
```

Comment:
Maybe try without comments @ the end of the Userstory :) 
also for the other Components

Reply:
Thank you for your remark! Removed the comments. :-)